recipes/sos-matlab/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sos-matlab" %}
+{% set version = "0.18.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54bf4ddcea30d43a90fc8a7432f990aae98351b9f0de4dc64196424ea1de99b9
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook >=0.19.4
```

Comment:
my comments are as before, I would add a hard dependency on python here. Maybe there should just be a run exports by python.

Reply:
just did it.